### PR TITLE
FAQ Page

### LINKED ISSUE #120: Create FAQ page with answers to common questions
To help new users understand Bespin, we should put together a list of FAQs and include this in the web application. Such questions should include:

- How do I get my data ready for Bespin?
- What kinds of pipelines are available?
- What if I need to change a parameter or use a different reference data set?
- Is there a limit on sizes for data?
- How long will my job take to run?
- How much will it cost?

### PR DESCRIPTION
**Pull request by dleehr**
Adds a FAQs page to display frequently asked questions and answers

- Created route `/faqs`, does not require authentication.
- Adds **FAQs** item to top navigation bar.
- FAQ content is markdown-formatted as an array of objects in `app/components/faqs/faq-list.js`.

Fixes #120

### CODE REVIEW

File: app/components/faqs/faq-list.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        'After the beta phase, the rate will be $0.04 per CPU hour. ' +
+        'The cost is calculated by multiplying this rate by the number of hours and number of CPUs used.'
+    },
+    {
+      question: 'How long does it take to run a workflow?',
+      answer:
+        'The time to run a workflow depends mostly on the input data size and resources (CPU, memory) assigned to the workflow. ' +
+        '\n\n' +
+        'For example, the Whole Exome Sequencing workflow uses 32 CPUs and can process 6 samples (75GB) in 23 hours, at a cost of about $30. ' +
+        '20 samples (250GB) can be processed in 72 hours, at a cost of about $90.'
```
Comment (johnbradley) [inline app/components/faqs/faq-list.js]: These two sentences confused me. On my laptop it wrapped like so:
```
... in 23 hours, at a cost of about $30. 20 
samples (250GB) can be processed in 72 hours, at a cost of about $90.
```
I read that as $30.20 for the first workflow and didn't realize the sentence changed. Perhaps starting with something like "The same workflow with"

File: app/components/faqs/faq-list.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    },
+    {
+      question: 'How do I use my data in Bespin?',
+      answer: 'Bespin can download your data from [Duke Data Service](https://dataservice.duke.edu). ' +
```
Comment (johnbradley) [inline app/components/faqs/faq-list.js]: `can download` makes me think there may be other ways to provide data (like urls or something). Perhaps we could reword this as a requirement?

File: app/components/faqs/faq-list.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      question: 'Are there limits to the size of a dataset Bespin can process?',
+      answer:
+        'Yes. Each job in Bespin has a limit of 10TB of input data. Beyond this point, another platform (e.g. an HPC cluster) ' +
+        'is likely a better fit for your project.'
+    },
+    {
+      question: 'What if I need to customize a parameter of the workflow or use a different set of reference data?',
+      answer:
+      'Great, please contact us [gcb-help@duke.edu](mailto://gcb-help@duke.edu). Bespin is designed to be highly ' +
+      'configurable, and we\'d be happy to see how we can make it work more investigators.'
```
Comment (johnbradley) [inline app/components/faqs/faq-list.js]: `work for more investigators`

File: app/components/faqs/faq-list.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  classNames: ['faq-list'],
+  faqs: [
+    {
+      question: 'What is Bespin?',
+      answer:
+        'Bespin is a web-based platform that makes it easy to run computational genomic workflows (pipelines) in the cloud.' +
+        '\n\n' +
+        'A service of the [Computational Solutions Shared Resource](https://genome.duke.edu/cores/computing) ' +
+        'in [Duke GCB](https://genome.duke.edu), Bespin offers provides best-practice workflows curated by experts. ' +
+        'Though the workflows are self-service, results are reviewed by experts.' +
```
Comment (johnbradley) [inline app/components/faqs/faq-list.js]: Who are these experts?

**Comment by hlapp** [comment]
> Do we need to include
> 
> * Questions related to Authorization Codes(job tokens) - like: Why do I need an Authorization Code before I can run a job?
> * Is this service HIPPA safe? (Not sure if the data would ever contain PHI)

Good suggestions!

**Comment by johnbradley** [comment]
Once I enter my username and password into the website `/faqs` redirects me to `/jobs`.

File: app/routes/faqs.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import Ember from 'ember';
+import UnauthenticatedRouteMixin from 'ember-simple-auth/mixins/unauthenticated-route-mixin';
+
+export default Ember.Route.extend(UnauthenticatedRouteMixin, {
```
Comment (johnbradley) [inline app/routes/faqs.js]: I think you need to remove the UnauthenticatedRouteMixin.
This only lets unauthenticated users see this page:
https://ember-simple-auth.com/api/classes/UnauthenticatedRouteMixin.html
